### ISSUE TITLE
cannot replicate your wonderful work

### ISSUE THREAD

**Issue by CCV-Edward** [kcct-fujimotolab/3DCNN#1]
Hello @kcct-fujimotolab
I am sorry to tell you that i can't replicate your fantastic work. I just input 'python 3dcnn.py --batch 32 --epoch 50 --videos dataset/ --nclass 10 --output 3dcnnresult/ --color True --skip False --depth 15',but i received low accuracy. To my surprise, this accuracy keep still, seems like update weights without better result....

**Comment by Lynte** [comment]
Thank you for the information.
Actually, when I got the result in result_3dcnn_10class, I executed like the following:
```sh
python 3dcnn.py --batch 128 --epoch 100 --videos dataset/ --nclass 10 --output result_3dcnn_10class --color False --skip False --depth 10
```
, so could you try this?

**Comment by CCV-Edward** [comment]
Thanks for your reply.
I executed as following:
`python 3dcnn.py --batch 128 --epoch 100 --videos dataset/ --nclass 10 --output result_3dcnn_10class --color False --skip True --depth 10`
,but I got following results:
`Epoch 1/100
128/963 [==>...........................] - ETA: 917s - loss: 14.1036 - acc: 0.12963/963 [==============================] - 1136s - loss: 14.0020 - acc: 0.1308 - val_loss: 14.1117 - val_acc: 0.1245`
`Epoch 2/100
963/963 [==============================] - 1127s - loss: 14.1096 - acc: 0.1246 - val_loss: 14.1117 - val_acc: 0.1245`
`Epoch 3/100
963/963 [==============================] - 1167s - loss: 14.1096 - acc: 0.1246 - val_loss: 14.1117 - val_acc: 0.1245`
`Epoch 4/100
963/963 [==============================] - 1256s - loss: 14.1096 - acc: 0.1246 - val_loss: 14.1117 - val_acc: 0.1245`
`Epoch 5/100
963/963 [==============================] - 1118s - loss: 14.1096 - acc: 0.1246 - val_loss: 14.1117 - val_acc: 0.1245`
`Epoch 6/100
963/963 [==============================] - 1130s - loss: 14.1096 - acc: 0.1246 - val_loss: 14.1117 - val_acc: 0.1245`
,it confuse me a lot. 
could you give me some advice?

**Comment by rysmarie** [comment]
I ran my code again and got results like this:
```
Epoch 1/100
1088/1088 [==============================] - 2s - loss: 2.3183 - acc: 0.1112 - val_loss: 2.2762 - val_acc: 0.0916
Epoch 2/100
1088/1088 [==============================] - 1s - loss: 2.1852 - acc: 0.2031 - val_loss: 2.0487 - val_acc: 0.2161
Epoch 3/100
1088/1088 [==============================] - 1s - loss: 2.0609 - acc: 0.2454 - val_loss: 1.9648 - val_acc: 0.2711
︙
```
I found the numbers of iterations are different(963 and 1088).
This can be occur if the names of the videos in your dataset are different from mine.
There is a possibility that the code fails to generate correct labels because of it.
Are their format same as 'v_HighJump_g18_c03.avi' ?

**Comment by CCV-Edward** [comment]
Thanks a lot! 
In term of numbers of iterations, it doesn't influence result.
Then, i download the UCF datasets, which has the same format as yours.
recently, i tried my best to overcome this problem , but still got unlucky result.

**Comment by rysmarie** [comment]
I executed the code in 3 computers but accuracy increases correctly. 
I used python3.5 (anaconda3-4.1.0) and Keras with tensorflow backend in all of the 3 computers.
Do other programs using Keras work correctly in your environment?

**Comment by CCV-Edward** [comment]
oh，I use python2.7!
Thanks a lot, it's other problem. With your help, I got a right answer.
